readium/navigator/src/main/java/org/readium/r2/navigator/media2/ExoPlayerFactory.kt

```diff
@@ -0,0 +1,56 @@
+package org.readium.r2.navigator.media2
+
+import android.content.Context
+import androidx.media2.common.SessionPlayer
+import com.google.android.exoplayer2.C
+import com.google.android.exoplayer2.ExoPlayer
+import com.google.android.exoplayer2.audio.AudioAttributes
+import com.google.android.exoplayer2.ext.media2.SessionPlayerConnector
+import com.google.android.exoplayer2.source.DefaultMediaSourceFactory
+import com.google.android.exoplayer2.upstream.DataSource
+import com.google.android.exoplayer2.upstream.cache.Cache
+import com.google.android.exoplayer2.upstream.cache.CacheDataSource
+import org.readium.r2.navigator.ExperimentalAudiobook
+import org.readium.r2.navigator.audio.PublicationDataSource
+import org.readium.r2.shared.publication.Publication
+import kotlin.time.Duration.Companion.seconds
+import kotlin.time.ExperimentalTime
+
+@ExperimentalAudiobook
+@OptIn(ExperimentalTime::class)
+class ExoPlayerFactory(private val cache: Cache? = null) {
+
+    fun createPlayer(context: Context, publication: Publication): SessionPlayer {
+        val dataSourceFactory = createDataSource(publication, cache)
+        val player: ExoPlayer = ExoPlayer.Builder(context)
+            .setSeekBackIncrementMs(30.seconds.inWholeMilliseconds)
+            .setSeekForwardIncrementMs(30.seconds.inWholeMilliseconds)
```

Comment:
Doesn't this conflict with the `MediaSessionNavigator.Configuration.skipForward/BackwardInterval` properties?

Reply:
No, actually this is useless since the corresponding commands are not used. I'll remove it to prevent confusion.